content/en/docs/concepts/configuration/secret.md

```diff
@@ -141,9 +178,13 @@ Each Secret you want to use needs to be referred to in `.spec.volumes`.
 If there are multiple containers in the Pod, then each container needs its
 own `volumeMounts` block, but only one `.spec.volumes` is needed per Secret.
 
-You can package many files into one secret, or use many secrets, whichever is convenient.
+{{< note >}}
+Secret volume sources are validated to ensure that the specified object
+reference actually points to an object of type Secret. Therefore, a secret
+needs to be created before any Pods that depend on it.
+{{< /note >}}
```

Comment:
please remove the 'note' shortcode unless we have a good reason to do so.

Reply:
I think this is a detail readers need to know. Is it not?

Reply:
Everything documented here is something readers need to know, but that doesn't mean we need to wrap them into a note.

Reply:
Fair enough!